lib/simplifyExpression/collectAndCombineSearch/ConstantOrConstantPower.js

```diff
@@ -38,7 +38,8 @@ function getExponentNode(node) {
 // Checks if the node is an constant or a power with constant base.
 // e.g. 2^3 is a constant power node, 5 is a constant node, x and x^2 are not
 function isConstantOrConstantPower(node) {
-  return ((NodeType.isOperator(node, '^') && NodeType.isConstant(node.args[0])) ||
+  return ((NodeType.isOperator(node, '^') &&
+          NodeType.isConstant(node.args[0])) ||
```

Comment:
ohhh yeah nvm you're right my bad

can you do this then?

```
(a &&
 b) ||
c
```

note the extra space to line a and b up together - idk it's still kinda hard to read but I guess it's ok :p